src/robots/RefreshRewardsRobot.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {IStaticATokenFactory} from '../interfaces/IStaticATokenFactory.sol';
+import {IRefreshRewardsRobot, AutomationCompatibleInterface} from '../interfaces/IRefreshRewardsRobot.sol';
+import {IStaticATokenLM} from '../interfaces/IStaticATokenLM.sol';
+import {IRewardsController} from 'aave-v3-periphery/contracts/rewards/interfaces/IRewardsController.sol';
+import {Ownable} from 'solidity-utils/contracts/oz-common/Ownable.sol';
+
+/**
+ * @title RefreshRewardsRobot
+ * @author BGD Labs
+ * @notice Automation contract to automate refresh rewards for staticATokens if a reward
+ *         is added after staticAToken creation to register the missing rewards.
+ */
+contract RefreshRewardsRobot is Ownable, IRefreshRewardsRobot {
+  mapping(address => bool) internal disabledStaticATokens;
+
+  /// @inheritdoc IRefreshRewardsRobot
+  IStaticATokenFactory public immutable STATIC_A_TOKEN_FACTORY;
+
+  /// @inheritdoc IRefreshRewardsRobot
+  IRewardsController public immutable REWARDS_CONTROLLER;
+
+  /// @inheritdoc IRefreshRewardsRobot
+  uint256 public constant MAX_ACTIONS = 10;
```

Comment:
is fine, but i think also a bit unnecessary in that case no?
Incentives need to be added via governance & there has rarely been a proposal adding two at once.

Might make sense to keep for consistency though as it doesn't hurt either.

Reply:
LM incentives are being added by the `EMISSION_ADMIN` currently, for a single reward for there can be many aTokens configured no? - for ex. on metis chain, metis token has been added as a reward for both aMetis and aWETH

Reply:
true